### DISCUSSION TITLE
Issue magick jpg to pdf

### DISCUSSION THREAD

**Discussion by Naoki031** [ImageMagick/ImageMagick#6956] [Digital Image Processing]
I am experiencing issues with version 7.1.1.22 on Alpine Linux. 
When I run the command 'magick file.jpg output.pdf. I'm unable to read the PDF file.
Error message: Failed to load PDF document.
Please, How to solve problem? 
Thank you

**Comment by snibgo** [comment]
> Error message: Failed to load PDF document.

What program did you use to read the PDF? Can Adobe Acrobrat Reader read it?

Please zip the input and output files, and post them here.
